team_test.go

```diff
@@ -762,14 +760,14 @@ func TestTeamAddContact(t *testing.T) {
 	// Arrange
 	testRequest := autopilot.NewTestRequest(
 		`mutation ContactCreate($input:ContactCreateInput!){contactCreate(input: $input){contact{address,displayName,id,type},errors{message,path}}}`,
-		`{"input": {"type":"slack", "address":"#general", "teamId": "{{ template "id1_string" }}" }}`,
+		`{"input": {"type":"slack", "address":"#general", "teamAlias": "{{ template "id1_string" }}" }}`,
```

Comment:
This should either remain as `teamId` or we shouldn't use this string template

Reply:
Great catch!! There was a related bug here and it's just been fixed.